.nvmrc

```diff
@@ -1 +1 @@
-14.19.0
+18.12.0
```

Comment:
Can you not simply put v18 so it uses the latest version instead of a fixed version?